lib/assets/javascripts/views/orgs/admin_edit.js

```diff
@@ -1,29 +1,67 @@
-// Import TinyMCE
-import tinymce from 'tinymce/tinymce';
-import { Tinymce } from '../../utils/tinymce';
 import ariatiseForm from '../../utils/ariatiseForm';
-import { SHOW_ORG_BANNER_MESSAGE } from '../../constants';
+import { Tinymce } from '../../utils/tinymce';
 
 $(() => {
   ariatiseForm({ selector: '#edit_org_details_form' });
 
-  // Returns text statistics for the specified editor by id
-  const getStats = (id) => {
-    const body = tinymce.get(id).getBody();
-    const text = tinymce.trim(body.innerText || body.textContent);
-    return {
-      chars: text.length,
-    };
+  // We only allow up to 3 URLs
+  const toggleAddUrlLink = () => {
+    if ($('#org-link-section').find('div.org-link').length >= 3) {
+      $('a#add-org-link').hide();
+    } else {
+      $('a#add-org-link').show();
+    }
+  };
+
+  // Remove a URL
+  const removeUrl = (e) => {
+    $(e.target).closest('.row').remove();
+    toggleAddUrlLink();
   };
 
-  // Validate banner_text area for less than 165 character
-  $('#edit_org_details_form').on('submit', (e) => {
-    if (getStats('org_banner_text').chars > 165) {
-      $('#org_banner_text').text(SHOW_ORG_BANNER_MESSAGE);
-      e.preventDefault();
+  const toggleFeedback = () => {
+    if ($('#org_feedback_enabled_true').is(':checked')) {
+      $('#feeback-email input, #feeback-email textarea').removeAttr('disabled');
+    } else {
+      $('#feeback-email input, #feeback-email textarea').attr('disabled', true);
     }
+  };
+
+  // Add a URL
+  $('a#add-org-link').click(() => {
+    const link = $('#org-link-section').find('div.org-link').last();
+    const clone = $(link).clone();
+    clone.find('input').val('');
+    $(clone).find('.remove-org-link').click((e) => {
+      removeUrl(e);
+    });
+    link.after(clone);
+    toggleAddUrlLink();
   });
-  /* Initialises an editor for textarea defined above with id org_banner_text */
-  Tinymce.init({ selector: '#org_banner_text' });
-});
 
+  $('.remove-org-link').click((e) => {
+    removeUrl(e);
+  });
+
+  $('#edit_org_feedback_form input[type="radio"]').click(() => {
+    toggleFeedback();
+  });
+
+  // Serialize URLs to JSON for form submission
+  $('#edit_org_profile_form').submit(() => {
+    const json = [];
+    $.each($('#org-link-section div.org-link'), (idx, e) => {
```

Comment:
If you use map for traversing is leaner since you need at the end that array of links JSON formatted